docs/content/en/docs-dev/user-guide/command-line-tool.md

```diff
@@ -311,6 +312,36 @@ You can encrypt it the same way you do [from the web](../managing-application/se
 
 Note: The docs for pipectl available command is maybe outdated, we suggest users use the `help` command for the updated usage while using pipectl.
 
+### Generating an application config (app.pipecd.yaml)
+
+Generate an app.pipecd.yaml interactively:
+
```

Comment:
[nit] It would be nice to add the link to the Feature Status page for `pipectl init` 👀 
So users can easily understand what kinds of apps are supported currently :)

Reply:
@ffjlabo 
Thanks!  I added some links!